flink-runtime/src/main/java/org/apache/flink/runtime/io/network/buffer/LocalBufferPool.java

```diff
@@ -357,9 +355,7 @@ public void lazyDestroy() {
 		}
 
 		// notify the potential blocking request thread
```

Comment:
we could remove this comment and add it in the javadoc of the method `mayNotifyAvailable`, then it can be reused by all the calls.